### ISSUE TITLE
can not run polynote when i choose python language version 0.2.8

### ISSUE THREAD

**Issue by jayeshd7** [polynote/polynote#531]
Hi Team,
I followed all steps which is mentioned and able to successful installed .(http://127.0.0.1:8192/)
but when i trying to run it is showing me below error code.
`    Uncaught exception: jep.Jep.init(Ljava/lang/ClassLoader;ZZ)J (java.lang.UnsatisfiedLinkError)

        polynote.server.repository.ipynb.JupyterOutput$.toResult(ast.scala:110)
        polynote.server.repository.ipynb.JupyterCell$$anonfun$toNotebookCell$2.apply(ast.scala:196)
        polynote.server.repository.ipynb.JupyterCell$$anonfun$toNotebookCell$2.apply(ast.scala:196)
        scala.collection.immutable.List.map(List.scala:284)
        polynote.server.repository.ipynb.JupyterCell$.toNotebookCell(ast.scala:196)
        polynote.server.repository.ipynb.JupyterNotebook$$anonfun$55.apply(ast.scala:248)
        polynote.server.repository.ipynb.JupyterNotebook$$anonfun$55.apply(ast.scala:248)
        scala.Function2$$anonfun$tupled$1.apply(Function2.scala:48)
        scala.Function2$$anonfun$tupled$1.apply(Function2.scala:47)
        scala.collection.immutable.List.map(List.scala:284)
        polynote.server.repository.ipynb.JupyterNotebook$.toNotebook(ast.scala:248)
        polynote.server.repository.ipynb.IPythonNotebookRepository$$anonfun$loadNotebook$1$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$4.apply(IPythonNotebookRepository.scala:35)
        polynote.server.repository.ipynb.IPythonNotebookRepository$$anonfun$loadNotebook$1$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$4.apply(IPythonNotebookRepository.scala:34)
        zio.ZIO$MapFn.apply(ZIO.scala:2682)
        zio.ZIO$MapFn.apply(ZIO.scala:2680)
        zio.internal.FiberContext.evaluateNow(FiberContext.scala:272)
        zio.internal.FiberContext.zio$internal$FiberContext$$run$body$2(FiberContext.scala:602)
        zio.internal.FiberContext$$anonfun$7.run(FiberContext.scala:602)
        java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        java.lang.Thread.run(Thread.java:748)`

please let me know is there any thing changes in template.yml file.

**Comment by jayeshd7** [comment]
JDK version is 1.8

**Comment by jeremyrsmith** [comment]
Seems like jep isn't being found. I *think* this is the same issue as #521 – can you take a look and report what you see when you run:

```
pip3 show jep
```

**Comment by jeremyrsmith** [comment]
Also, welcome and thanks for the issue report! 😄

**Comment by jayeshd7** [comment]
insted of pip3 , I am using pip.
when i ran above command-
below is the output


(base) 90-X50XLVDQ-3GN:Applications jayeshdilip.dalal$ pip show jep
Name: jep
Version: 3.9.0
Summary: Jep embeds CPython in Java
Home-page: https://github.com/ninia/jep
Author: Jep Developers
Author-email: jep-project@googlegroups.com
License: zlib/libpng
Location: /anaconda3/lib/python3.7/site-packages
Requires: 
Required-by:

**Comment by jonathanindig** [comment]
Hi @jayeshd7 can you try launching Polynote with this script: https://github.com/polynote/polynote/blob/master/scripts/polynote.py

Just copy it to your polynote directory and run it. We think this script should solve some of the problems people have been running into with Jep and it will be the default next release.

**Comment by austinmw** [comment]
@jonathanindig I'm still getting an error `Uncaught exception: jep.Jep.init(Ljava/lang/ClassLoader;ZZ)J (java.lang.UnsatisfiedLinkError)`.

I used that script, but replaced `#!/usr/bin/env python3` with `#!/usr/local/anaconda3/bin/python` (a conda install of python v3.7)

**Comment by jonathanindig** [comment]
Welcome @austinmw, thanks for the report. Do you have the `$PYTHONHOME` environment variable set? Can you try setting it to your anaconda installation directory (maybe `/usr/local/anaconda3/`)? 

If that doesn't work, perhaps something in the rest of this FAQ might help? https://github.com/ninia/jep/wiki/FAQ#how-do-i-fix-unsatisfied-link-error-no-jep-in-javalibrarypath

**Comment by austinmw** [comment]
@jonathanindig Hi, yes I tried `PYTHONHOME=/usr/local/anaconda3/` and ``PYTHONHOME=/usr/local/anaconda3/bin/python` in both bash and zsh but always get the same error.

**Comment by simonohanlon101** [comment]
@austinmw I solved this issue for using polynote with a `conda` environment by:
1) remove the shebang line at the top of the `polynote.py` file - we do not want this script to be run by `#!/usr/bin/env python3` but by the interpreter of our conda environement.
2) Set `PYTHONHOME` to the location of the `conda` environment. For me, if I run `conda create -n polynote` the environment lives here: `/anaconda3/envs/polynote` so I ran: `EXPORT PYTHONHOME=/anaconda3/envs/polynote`
3) Run the script using the activated conda environment's python, i.e. `python polynote.py` (if you aren't sure: `which -a python` - I'm also assuming you installed the other dependencies _in_ the environment)
